code/modules/surgery/limb_augmentation.dm

```diff
@@ -43,6 +43,11 @@
 	possible_locs = list(BODY_ZONE_R_ARM,BODY_ZONE_L_ARM,BODY_ZONE_R_LEG,BODY_ZONE_L_LEG,BODY_ZONE_CHEST,BODY_ZONE_HEAD)
 	requires_real_bodypart = TRUE
 
+
+
+/datum/surgery/augmentation/can_start(mob/user, mob/living/carbon/target)
+	return !isoozeling(target)
```

Comment:
```suggestion
/datum/surgery/augmentation/can_start(mob/user, mob/living/carbon/target)
	return ..() && !isoozeling(target)
```

Reply:
@park66665 Would parent call be needed for this? most non-advanced surgeries don't call

Reply:
`/datum/surgery/proc/can_start` is actually a reasonably complex set of checks which has been in use for augmentation surgery. I wouldn't override that.